instagram/src/components/PostContainer/Post.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import CommentSection from '../CommentSection/CommentSection';
+import PostHeader from './PostHeader'; 
+import HeartMessageIcons from './HeartMessageIcons';
+
+class Post extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            likes: props.post.likes,
+
+            color: 'black'
+        };
+    }
+
+    increaseLikes = _ => {
+        let likes = this.state.likes;
+
+        if(this.state.color === 'black') {
+            this.setState({ likes: likes + 1, color: 'red' });
+        } else {
+            this.setState({ likes: likes - 1, color: 'black' });
```

Comment:
you might want to go with a boolean for this - for whether it's liked or not.  but this works, so good job!